base/nsjail-docker/Dockerfile

```diff
@@ -30,9 +30,12 @@ RUN python3 -m venv /venv
 RUN bash -c "source /venv/bin/activate && pip3 install ecdsa"
 RUN mv /venv /chroot/
 
+#TODO should these exist by default? Or should the chal author specify mappings manually?
```

Comment:
@sirdarckcat what do you think about this?
Should we stop mounting these directories by default and have the challenge author specify all mounts themselves?
Pro: no accidental leaks of files into the chroot
Pro: a cleaner chroot, everything is where it should be (flag at /flag for example)
Contra: extra step when adding a new file

Reply:
I imagine it'll be confusing for most authors to make sense of whats gonna get mounted by k8s, so I think it's better if we do it for them as otherwise they won't use it

Reply:
ok, moved it to the challenge-skeleton dockerfile though since it's not needed by the base and users might want to clean it up.